src/mon/PGMonitor.h

```diff
@@ -38,10 +38,13 @@ class MGetPoolStats;
 
 class TextTable;
 class MPGStats;
+class PGStatService;
+class PGMapStatService;
 
 class PGMonitor : public PaxosService {
 public:
   PGMap pg_map;
+  PGMapStatService *pgservice;
```

Comment:
Right.
So you'll see the MMonMgrReport is empty right now (huh, I thought I had it just sending a whol PGMap...weird!), but I want to have that submit all the information we actually need for the pgservice interface, and the MgrPGStatService ingests that to provide the data actually needed. Then we can turn off as much of the pgmonitor as is feasible (for luminous) — should be most of it since only the message dispatch pipeline directly refers to the PGMonitor any more?

Once we're done with Luminous, we should be able to rip everything out pretty easily since it's hidden behind the interfaces.